File: 1236-n-th-tribonacci-number/n-th-tribonacci-number.go
func tribonacci(n int) int {
    var counter int = 3
    var a int = 0
    var b int = 1
    var c int  = 1

    if n ==0{
        return 0
    }
    if n==1{
        return 1
    }

    if n==2{
        return 1
    }
    if n==3{
        return 2
    }
    for counter<=n{
        var d int = a+b+c
        fmt.Println(d)
        a = b
        b=c
        c=d
        counter +=1
    }
    return c
    
}

File: 1236-n-th-tribonacci-number/n-th-tribonacci-number.py
class Solution:
    def tribonacci(self, n: int) -> int:
        a = 0
        b = 1
        c = 1
        if n==0:
            return 0
        if n ==1:
            return 1
        if n == 2:
            return 1
        if n== 3:
            return 2
        for i in range(3,n+1):
            d = a+b+c
            a = b
            b = c
            c = d
        
        return c

        